src/dcap_provider.cpp

```diff
@@ -1729,6 +1738,7 @@ extern "C" quote3_error_t sgx_ql_get_qve_identity(
 }
 
 extern "C" quote3_error_t sgx_ql_get_root_ca_crl(

```

Comment:
I don't think you can change the signature of this method.. as it is part of the QPL specification:
https://download.01.org/intel-sgx/sgx-dcap/1.5/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf

Reply:
This seemed like the easiest way to test for a secondary root crl. I'll revert it I guess as it was diagnostic. I just figured since we're leaving the sbx changes in this was fine.

Reply:
SBX root crl will be tested in `GetVerificationCollateralTestICXV3()` as `sgx_ql_get_quote_verification_collateral("platform", ...)` will fetch root crl from SBX endpoint. The returned `collateral` will also be tested by `VerifyCollateral()`.